recipes/oauthenticator/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "oauthenticator" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f174c7b2fa20028717aa10e778f66ffa896a2fbcb2ee8a78b135d5aa8a5d56ff
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - jupyterhub >= 0.5
+
+test:
+  imports:
+    - oathenticator
+
+about:
+  home: https://github.com/jupyterhub/oauthenticator
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'OAuth + JupyterHub Authenticator = OAuthenticator'
+
+  dev_url: https://github.com/jupyterhub/oauthenticator
```

Comment:
Can you add the `doc_url` as well? I think they keep their documentation here: https://jhubdocs.readthedocs.io/en/latest/oauthenticator

Reply:
done